src/ressources/customers/FormattedTotalSpentField.tsx

```diff
@@ -0,0 +1,23 @@
+import { NumberField, useRecordContext } from "react-admin";
+
+type FormattedTotalSpentFieldProps = {
+  source: string;
+};
+
+export const FormattedTotalSpentField = ({
+  source,
+}: FormattedTotalSpentFieldProps) => {
+  const record = useRecordContext();
+
+  return record ? (
+    <NumberField
+      color={() => (record[source] > 500 ? "red" : "grey")}
+      source={source}
+      label="Total spent"
+      options={{ style: "currency", currency: "USD" }}
+    />
+  ) : null;
+};
+FormattedTotalSpentField.defaultProps = {
+  textAlign: "right",
+};
```

Comment:
Do you actually use this? 🤔

Reply:
Maybe I can put it somewhere else, this is the result without it :
![image](https://user-images.githubusercontent.com/100569076/164025730-59e006c7-e4c6-4171-85dc-d5a847c35e4f.png)